packages/components/src/components/textarea/textarea.ts

```diff
@@ -1,23 +1,22 @@
-import { classMap } from 'lit/directives/class-map.js';
-import { customElement } from '../../../src/internal/register-custom-element';
-import {property, query, state } from 'lit/decorators.js';
-import { defaultValue } from '../../internal/default-value';
-import { FormControlController } from '../../internal/form';
-import { HasSlotController } from '../../internal/slot';
-import { html } from 'lit';
+import { css, html } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { defaultValue } from '../../internal/default-value.js';
+import { FormControlController } from '../../internal/form.js';
+import { HasSlotController } from '../../internal/slot.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { live } from 'lit/directives/live.js';
-import { watch } from '../../internal/watch';
+import { property, query, state } from 'lit/decorators.js';
+import { watch } from '../../internal/watch.js';
+import cx from 'classix';
+import formControlStyles from '../../styles/form-control.styles';
 import SolidElement from '../../internal/solid-element';
-import styles from './textarea.styles';
-import type { CSSResultGroup } from 'lit';
 import type { SolidFormControl } from '../../internal/solid-element';
 
 /**
  * @summary Textareas collect data from the user and allow multiple lines of text.
- * @documentation https://solid.union-investment.com/[storybook-link]/textarea
+ * @documentation https://shoelace.style/components/textarea
  * @status stable
- * @since 1.0
+ * @since 2.0
```

Comment:
```suggestion
 * @since 1.30.0
```

I believe this is the correct version if this is the next component to be merged.

Reply:
Thanks  for the hint, the 2 comes still from the shoelace repository. How does the versioning in solid works? I just had a look at a few different component and couldn't see a pattern:
accordion
 * @since 1.1

button
  * @since 1.0

badge
 * @since 1.6.0

Reply:
Oh I see. I take a look at the most recent version number on the repository, and bump it one version on the component I am merging.

<img width="350" alt="image" src="https://github.com/solid-design-system/solid/assets/63849626/d46c68a4-0ae0-4d79-a005-5be7e6d32e8f">

But it would be great if someone else from @solid-design-system/core-development also confirm this :)